src/components/button/ButtonGroup.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import styles from './buttonGroup.module.scss';
+import { useNavigate } from 'react-router-dom';
+
+interface ButtonGroupProps {
+  onCancel?: () => void;
+  onSubmit?: () => void;
+}
+
+const ButtonGroup: React.FC<ButtonGroupProps> = ({ onCancel, onSubmit }) => {
+  const navigate = useNavigate();
+
+  const handleCancel = () => {
+    if (onCancel) {
+      onCancel();
+    } else {
+      navigate(-1); //기본 동작 -> 이전페이지 이동
+      console.log('취소 버튼 클릭');
+    }
+  };
```

Comment:
if else 문보다 early return 문을 사용하시는 건 어떠신가용
```suggestion
    if (onCancel) {
      return onCancel();
    }
     navigate(-1); //기본 동작 -> 이전페이지 이동
     console.log('취소 버튼 클릭');
  };
```

Reply:
early return문 적용했슴니당!